index.js

```diff
@@ -4,21 +4,25 @@
  * @function initLoginButton
  * @param {String} containerId The id property of the container element to place the login button / status in
  * @param {String} baseUrl The base URL of the MIT Open instance
+ * @param {String} redirectUrl The URL to redirect to after login is complete
  * @param {String} buttonText The text to show on the Login button
  * @param {String} buttonClass The CSS class(es) to assign to the button
  * @param {String} loggedInTextClass The CSS class(es) to assign to the logged-in status text
  */
 export function initLoginButton(
   containerId,
   baseUrl,
+  redirectUrl = "",
   buttonText = "Login",
   buttonClass = "",
   loggedInTextClass = "",
 ) {
   const container = document.getElementById(containerId)
   const parsedBaseUrl = new URL(baseUrl)
   const apiUrl = `${parsedBaseUrl.origin}/api/v0/users/me?format=json`
-  const loginUrl = `${parsedBaseUrl.origin}/login/ol-oidc/`
+  const redirectUrlParam =
+    redirectUrl !== "" ? `?redirect_url=${redirectUrl}` : ""
```

Comment:
I think this needs to be encoded:

```suggestion
    redirectUrl !== "" ? `?redirect_url=${encodeURIComponent(redirectUrl)}` : ""
```

Reply:
Thanks, I implemented your suggestion